runner/Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM rust:latest
-RUN rustup install nightly
+VOLUME /app/src
+VOLUME /app/benches
+VOLUME /app/inputs
+VOLUME /app/target
```

Comment:
We have to do this because `/app` is now mixed between container-owned (/app/vendor) and host-owned (the volumes).